### PR TITLE
fix(@chat): only show reaction to emoji right click

### LINKED ISSUE #5764: edit messages context menu should not appear for emoji messages
# Bug Report

## Steps to reproduce

1. send a message with a single emoji
2. click right on the message

result: "edit message" option appears in context menu
expected result: "edit message" option should not appear for emoji messages

### PR DESCRIPTION
**Pull request by alaibe**
fixes #5764
image

### CODE REVIEW

File: ui/imports/shared/views/chat/CompactMessageView.qml [hunk 1]
```diff
@@ -102,9 +102,9 @@ Item {
         isHovered: root.isHovered
         isMessageActive: root.isMessageActive
         isActivityCenterMessage: activityCenterMessage
-        stickersLoaded: root.stickersLoaded
+        stickersLoaded: root.stickersLoadede.g
```
Comment (saledjenic) [inline ui/imports/shared/views/chat/CompactMessageView.qml:105]: @alaibe is this a typo here?

**Review by saledjenic — APPROVED** [review]

**Comment by anastasiyaig** [comment]
i think editing emoji is fine, but editing a sticker is something we dont want to edit _i guess_ . To be sure, we need to check on mobile :) And make it like that :)

File: ui/imports/shared/views/chat/CompactMessageView.qml (hunk 1 above)
Comment (alaibe) [reply ui/imports/shared/views/chat/CompactMessageView.qml]: :eagle: super well spotted!

**Comment by saledjenic** [comment]
As I see here in the title of corresponding ticket #5764 it says "edit messages context menu should not appear for emoji messages", so this PR is about emoji.

**Comment by alaibe** [comment]
@saledjenic @anastasiyaig I am also not sure to really understand such "extra" complexity which I think doesn t really provide a value. As Sale said: an emoji is a text message.

But this is what the issues says.
Also I asked in desktop-ui what are the expected context menu for emoji

**Comment by anastasiyaig** [comment]
i agree with u @alaibe , but i checked in mobile - they dont allow editing emoji only message too. Even though i am not clear on why it is so, i believe we need to be aligned on that with them

**Comment by richard-ramos** [comment]
@anastasiyaig, @alaibe 
The reason why only text messages are allowed to be edited and not emoji messages / stickers, is because all these messages have different ContentType and attributes, while normal messages have `TEXT_PLAIN` and `text` attribute and at the time it was the simplest solution that could be done to demonstrate the editing functionality. If we were to support other type of messages, we could modify status-go to do something like this: instead of accepting a `{"id":"the_message_id", "text": "the new text"}`, it should accept a json with different attributes:: 
```js
{
    "id":"the_message_id", 
    "contentType":   1.... // the new content type of the message

    // optional, only if the content type is plain text,  or emoji
    "text": "the new text" 

     // optional, only if the content type is sticker
    "packId": "the pack id"
    "hash": "the sticker hash"

   // etc etc for other content types...
}
```
I'm not sure what implications this would have for status-react / status-desktop since the edited message type could change

**Review by anastasiyaig — APPROVED** [review]

**Pull request merged**